pkg/util/containers/providers/windows/provider.go

```diff
@@ -155,8 +156,31 @@ func (mp *provider) GetContainerMetrics(containerID string) (*metrics.ContainerM
 
 // GetContainerLimits returns CPU, Thread and Memory limits
 func (mp *provider) GetContainerLimits(containerID string) (*metrics.ContainerLimits, error) {
-	// FIXME: Figure out a way to extract limits from Job Objects in the Windows Kernel
-	return nil, fmt.Errorf("not supported on windows")
+	dockerUtil, err := docker.GetDockerUtil()
+	if err != nil {
+		return nil, err
+	}
+
+	cjson, err := dockerUtil.Inspect(containerID, false)
+	if err != nil {
+		return nil, err
+	}
+
+	var cpuMax float64 = 0
+	if cjson.HostConfig.NanoCPUs > 0 {
+		cpuMax = float64(cjson.HostConfig.NanoCPUs) / 1e9 / float64(sysinfo.NumCPU()) * 100
+	} else if cjson.HostConfig.CPUPercent > 0 {
+		cpuMax = float64(cjson.HostConfig.CPUPercent)
+	} else if cjson.HostConfig.CPUCount > 0 {
+		cpuMax = float64(cjson.HostConfig.CPUCount) / float64(sysinfo.NumCPU()) * 100
```

Comment:
Should be min(cjson.HostConfig.CPUCount, sysinfo.NumCPU()) / sysinfo.NumCPU() as `CPUCount` is not bounded